server/README.md

```diff
@@ -0,0 +1,139 @@
+# Language server bitbkae
+
+Language server for bitbake.
+
+![screenshot](https://github.com/yoctoproject/vscode-bitbake/assets/32936898/82e16428-e5fb-4e37-a2fa-b56c00cc4f4c)
+
+## Install
+
+```sh
+npm install -g language-server-bitbake
+```
+
+## Configure
+
+- For windows, change `~/.config` to `~/AppData/Local`
```

Comment:
where should this be changed? I think you could explain:
```
In the following instructions, replace `~/.config` to `~/AppData/Local` for Windows.
```

Reply:
OK, Fixed.